src/constants/Links.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+function PublicLinks() {
+  return (
+    <div className="navbar">
+      <ul>
+        <li>
+          <NavLink to="/">Home</NavLink>
+        </li>
+        <li>
+          <NavLink to="/login">Login</NavLink>
+        </li>
+      </ul>
+    </div>
+  );
+}
+
+export default PublicLinks;
```

Comment:
research on a better way to do this, you just copy pasted the navbar component

Reply:
Okay